lambdas/indexer/app.py

```diff
@@ -151,7 +173,52 @@ def update_health_cache(_event: chalice.app.CloudWatchEvent):
     app.health_controller.update_cache()
 
 
-@app.route('/{catalog}/{action}', methods=['POST'])
+@app.route('/{catalog}/{action}', methods=['POST'], method_spec={
+    'tags': ['Indexing'],
+    'summary': 'Notify the indexer to perform an action on a bundle',
+    'description': format_description('''
+        Queue a bundle for addition to or deletion from the index. If the
+        request body contains a valid bundle notification, the notification is
+        sent to an SQS queue that will deliver it to an indexer Lambda function.
+        The request must be authenticated using HMAC via the ``signature``
+        header. Each Azul deployment has its own unique HMAC key. The HMAC
+        components are the request method, request path, and the SHA256 digest
+        of the request body.
+    '''),
+    'requestBody': {
+        'description': 'Contents of the notification',
+        'required': True,
+        **json_content(schema.object(
+            bundle_fqid=schema.object(
+                uuid=str,
+                version=str,
+                source=schema.object(
+                    id=str,
+                    spec=str
+                )
+            )
+        ))
+    },
+    'parameters': [
```

Comment:
I thought there was a way to declare headers in Swagger. If that is correct it should be used to document the HMAC header.